src/interface/rangePicker/RangePicker.ts

```diff
@@ -0,0 +1,13 @@
+export interface RangePickerInterface{
+    sliderLength?: number,
+    min?: number,
+    max?: number,
+    step?: number,
+    buttonBorderRadius?: number,
+    backgroundColor?: string,
+    color?: string,
+    trackStyling?:any,
+    SelectedStyling?:any,
```

Comment:
change any to object. Please update output here

Reply:
resolved